keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -1026,7 +1057,8 @@ kpxc.initCredentialFields = async function(forceCall, inputs) {
 
     _called.clearLogins = true;
 
-    if (kpxc.siteIgnored()) {
+    // No need to check this for every dynamically added inputs
+    if (inputs === undefined && kpxc.siteIgnored()) {
```

Comment:
Again should be ||

Reply:
The whole point of this check is to use `kpxc.siteIgnored()` only when `kpxc.initCredentialFields()` is called without dynamically added input fields. But now when I think of it, the whole site ignored check should be done outside of this function when the page loads, not with every possible check. So making a fix.

Reply:
Good point! Even better